### PR TITLE
Add/onboarding wordpress importer

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

* Registered a WordPress onboarding flow `/start/from/importing/wordpress`
* Added feature flag `gutenboarding/import-from-wordpress`
* Added a *static* content type selector screen

#### Testing instructions

* go to `/start/from/importing/wordpress?from={WP_SITE_URL}&to={slug}.wordpress.com`
* check if there is a static "content type selector" screen

#### Screenshots

**Desktop**
Screenshot 2021-12-23 at 14 23 54

**Mobile**
Screenshot 2021-12-23 at 14 34 51

Related to #59042

### CODE REVIEW

File: client/signup/steps/import-from/wordpress/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+				<div className={ 'step-wrapper__header-image' }>
+					<img alt="" src="/calypso/images/importer/onboarding-2.svg" />
+				</div>
+			</div>
+			<div className={ 'import-layout__column' }>
+				<div>
+					<ActionCard
+						classNames={ classnames( 'list__importer-option', { 'is-disabled': true } ) }
+						headerText={ __( 'Everything' ) }
+						mainText={ __( "Your site's content, themes, plugins, users and settings" ) }
```
Comment (a8ci18n) [inline client/signup/steps/import-from/wordpress/index.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'All your site's content, themes, plugins, users and settings' )` *ES Score: 9*

Comment (bogiii) [reply client/signup/steps/import-from/wordpress/index.tsx]: I would reuse the already translated string ☝️. The only difference is the beginning of a sentence.
Do you agree, @vishnugopal?

File: client/signup/steps/import-from/wordpress/content-chooser/index.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+				<div className={ 'step-wrapper__header-image' }>
+					<img alt="" src="/calypso/images/importer/onboarding-2.svg" />
+				</div>
+			</div>
+			<div className={ 'import-layout__column' }>
+				<div>
+					<ActionCard
+						classNames={ classnames( 'list__importer-option', { 'is-disabled': true } ) }
+						headerText={ __( 'Everything' ) }
+						mainText={ __( "Your site's content, themes, plugins, users and settings" ) }
```
Comment (a8ci18n) [inline client/signup/steps/import-from/wordpress/content-chooser/index.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'All your site's content, themes, plugins, users and settings' )` *ES Score: 9*

File: client/signup/steps/import-from/wordpress/index.tsx (hunk 1 above)
Comment (vishnugopal) [reply client/signup/steps/import-from/wordpress/index.tsx]: Yes agree!

**Review by vishnugopal — APPROVED** [review]
Works well for me!

File: client/signup/steps/import-from/wordpress/index.tsx (hunk 1 above)
Comment (zaerl) [reply client/signup/steps/import-from/wordpress/index.tsx]: 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7057194

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7057194).

**Pull request merged**